### ISSUE TITLE
Does charm++ checkpointing with charm 7, latest develop work?

### ISSUE THREAD

**Issue by geoffrey4444** [sxs-collaboration/spectre#4595]
We'll need to be able to checkpoint and restart to run an inspiral to common horizon. Last time @geoffrey4444 checked, checkpointing using charm++ checkpointing did not work well: each time restarting required more memory, and took longer, than the last. Perhaps the issues from before have been resolved.

If it works, great. If not, then @nilsdeppe and @pajkosmi will look into checkpointing manually.

**Comment by knelli2** [comment]
Yes, charm++ checkpoints works well! I did two different types of runs (on Wheeler).

1. Checkpoint periodically during a single evolution. I ran for 20M and checkpointed every 1M, so I had 20 checkpoints total. The size of the checkpoints remained the same for all checkpoints.
2. Checkpoint before the wallclock, stop the evolution, submit a new job, and restart from the checkpoint. I also did this 20+ times and the checkpoints were all the same size. And they were the same size as in 1. also.

Both of these results give me confidence that charm++ checkpoints should just work for restarting an evolution from the exact same parameters.
